test/GoTrueClient.test.ts

```diff
@@ -280,6 +281,19 @@ describe('GoTrueClient', () => {
     expect(user?.email).toBe(email)
   })
 
+  test('signIn with OpenIDConnect', async () => {
+    const openIDConnect:OpenIDConnectCredentials = {
+      IDToken: 'abcde',
+      nonce: 'random value',
```

Comment:
Is there a suggested way to generate a nonce?

Should the GoTrue client offer up a `generateOidcNonce()` method?

Reply:
generateOidcNonce() is not required.
Please read the oidc specification carefully.

The ability to suggest passwords to end users is not in the scope of supabase.
Similarly, the ability to suggest a nonce to each and every developer should not be in the scope of supabase.